### DISCUSSION TITLE
npm published versions

### DISCUSSION THREAD

**Discussion by yosiat** [nanoexpress/nanoexpress#344] [Q&A] [answered]
Hi!

I don't see versions greater than v5.1.2 in npm, but I see in github lots of tags until v8.0.0.
Is there any specific reason why it wasn't published to npm?

**Comment by dalisoft** [comment, marked as answer]
@yosiat Thank you for interest. I didn’t published to npm because of these releases aren’t stable and some of users could upgrade and break their production backend
